helm/chaos-mesh/values.yaml

```diff
@@ -70,6 +70,18 @@ chaosDaemon:
   # runtime: containerd
   # socketPath: /run/containerd/containerd.sock
 
+  resources:
```

Comment:
Should we remove these resource limits rather than add them for all? @cwen0

Reply:
I agree with you. I think `resources:{}` is OK.